File: Java/src/com/ecc/gkgo1a/K08_1Sub.java
package com.ecc.gkgo1a;

class K08_1Sub {
    static int GetMaxValue(int v1, int v2, int v3) {
        int temp = v1 >= v2 ? v1 : v2;
        temp = temp >= v3 ? temp : v3;
        return temp;
    }

    static int GetMinValue(int v1, int v2, int v3) {
        int temp = v1 < v2 ? v1 : v2;
        temp = temp < v3 ? temp : v3;
        return temp;
    }
}

File: Scala/src/main/scala/PrimaryClasses/K08_1sub.scala
package PrimaryClasses

object K08_1sub {
  def GetMaxValue(v1: Int, v2: Int, v3: Int): Int = {
    var temp = if (v1 >= v2) v1 else v2
    temp = if (temp >= v3) temp else v3
    temp
  }

  def GetMinValue(v1: Int, v2: Int, v3: Int): Int = {
    var temp = if (v1 < v2) v1 else v2
    temp = if (temp < v3) temp else v3
    temp
  }
}
